tests/unit/drivers/test_reduce_all_driver.py

```diff
@@ -67,3 +68,27 @@ def validate(req):
         flow.index(input_fn=input_fn, on_done=response_mock)
 
     response_mock.assert_called()
+
+
+def test_reduce_all_root_chunks(mocker, docs):
```

Comment:
This should go with best possible effort into an `integration` test. Try not to add tests with `Flow` in unittests

Reply:
It can be done in another PR though

Reply:
While I see your point, this was here before. Not part of this PR for now, to not obfuscate more, what actually happens in this PR.